unturned/OpenMod.Unturned/Events/UnturnedPlayerEventsListener.cs

```diff
@@ -1,7 +1,8 @@
-﻿using SDG.Unturned;
+﻿extern alias MonoModILHelpers;
+using MonoModILHelpers::System.Runtime.CompilerServices;
```

Comment:
Don't use `extern alias` of monomod, use the .net standard API

Reply:
Try to remove it, you will get an error due to multiple Unsafe types with the same name and namespace

Reply:
If you will try to change alias name to system (i.e netstandard) it won't work, idk why

Reply:
Fixed